fms/models/gpt_bigcode.py

```diff
@@ -280,6 +285,13 @@ def __init__(
         self.config = self.config.updated(**kwargs)
         self.distributed_strategy = distributed_strategy
 
+        gptq_config_dict = kwargs.get('gptq_config', None)
```

Comment:
I'd change this to a field called `linear_config` instead, and you can make it a part of the GPTBigCodeConfig if needed (and Llama, etc.). This way we make it more general and less GPTQ dpeendent